addons/mrp/models/mrp_production.py

```diff
@@ -289,25 +290,20 @@ def _compute_mrp_production_backorder(self):
         for production in self:
             production.mrp_production_backorder_count = len(production.procurement_group_id.mrp_production_ids)
 
-    @api.depends('move_raw_ids', 'state')
+    @api.depends('move_raw_ids', 'state', 'date_planned_start', 'move_raw_ids.forecast_availability', 'move_raw_ids.forecast_expected_date')
     def _compute_components_availability(self):
         self.components_availability = False
         self.components_availability_state = 'available'
         productions = self.filtered(lambda mo: mo.state not in ['cancel', 'draft', 'done'])
+        productions.components_availability = _('Available')
```

Comment:
Move to assignation in a `else` at the end of the `for` loop

Reply:
Too slow :p